src/config/configMaps.js

```diff
@@ -45,9 +45,9 @@ export const search = {
   sortSearchByDistance: false,
 
   // Limit location autocomplete to a one or more countries
-  // using ISO 3166 alpha 2 country codes separated by commas.
+  // using CLDR two-character region codes, separated by commas.
   // If you want to limit the autocomplete, uncomment this value:
-  // countryLimit: ['AU'],
+  // countryLimit: ['us'],
```

Comment:
Don't change this.

Did you try capital letters? If they work, keep it as they are. If not, then those values, need to be converted to lower case before they are given to Google Maps.